libs/jsonschema/schema_test.go

```diff
@@ -250,3 +256,52 @@ func TestValidateSchemaMinimumCliVersion(t *testing.T) {
 	err = s.validateSchemaMinimumCliVersion("v0.0.0-dev")()
 	assert.NoError(t, err)
 }
+
+func TestValidateSchemaConstTypes(t *testing.T) {
+	s := &Schema{
+		Properties: map[string]*Schema{
+			"foo": {
+				Type:  "string",
+				Const: "abc",
```

Comment:
What are valid types for Const? Only strings and integers?

Reply:
Const can be any type, including null or objects according to the JSON schema spec: https://json-schema.org/draft/2020-12/json-schema-validation#name-const

Reply:
Right now we only validate the const keyword, because that all we need for the mlops-stacks requirements.

Reply:
@shreyas-goenka what if I define the following const, does it work?

```
"properties": {
        "def": {
            "type": "float",
            "const": 5.1
        }
    }
```

Reply:
Tried it out. Does not work for float because of precision issues. (5.1 vs 5.099). 

Also does not work for integers due to type not being correct (basically const is parsed as a float rather than an int). I can fix it in a followup PR.